packages/@lightningjs/ui-components/docs/Theming.stories.mdx

```diff
@@ -53,7 +53,7 @@ Each of the components references specific style properties that map back to a g
 To start applying themes to your components you need to import both `context` and the theme you want to use in your file, if different from the Base theme. If you do not import a theme, you will default to the open-source Base theme.
 
 ``` js
-import { context } from @lightningjs/ui;
+import { context } from '../../globals';
```

Comment:
I actually think this is supposed to say `import { context } from '@lightningjs/ui-components';` since it's example documentation